woocommerce/woocommerce-ios
# [Mobile Payments] Clean up feature flags

## Description


This PR removes a number of payments-related feature flags, for changes which were previously released.

It's not urgent, so I've tagged it for 16.2.

I've split the removals up into individual commits, which may be an easier way to review this.

## Testing instructions


Check that unit tests pass.

Check the following features work as expected:

- [ ] For a new install of the app, when completing IPP onboarding you're prompted to enable Pay in Person
- [ ] For a Woo Express free trial site with a US app store account, In-App Purchases are offered for upgrades
- [ ] Just in Time Messages are shown on the dashboard (test without the throttle, you'll need a sandbox)
- [ ] IPP feedback banner is shown on the Order List
- [ ] Retrying a failed IPP works
- [ ] Discounts can be added to a product in an order, and the UX is as expected
- [ ] In person payments for orders work, and a reload of the order is done when the payment is started
- [ ] Manual taxes work as expected
- [ ] Do a release build to a simulator, check in `Menu > Settings > Experimental Features` – an `In App Purchases` toggle should **not** be visible.


---
